spec/tictactoe_clojure/game_spec.clj

```diff
@@ -0,0 +1,16 @@
+(ns tictactoe-clojure.game-spec
+  (:require [speclj.core :refer :all]
+            [tictactoe-clojure.console :as console]
+            [tictactoe-clojure.game :refer :all]))
+
+(describe "tictactoe-clojure.game"
+  (with new-board [1 2 3 4 5 6 7 8 9])
+  (with first-player #tictactoe_clojure.player.Human{:piece "X"} )
+  (with second-player #tictactoe_clojure.player.Human{:piece "O"})
+  (with players [@first-player @second-player])
+
+  (describe "#play-game"
+    (it "loops through a game where the human enters 1, 2, 4 and loses to the computer"
+      (with-redefs [console/display (fn [_] "Player O wins")]
+      	(with-out-str (with-in-str "1\n4\n2\n5\n3\n"
+          (should= "Player O wins" (play-game @players @new-board))))))))
```

Comment:
Here's what I would test for:

1. that your `console/display` is called with "Player [X or O] move\n", 
2. that you display your board (you dont have to make it spit out the actual one)
3. what happens when the game is over?
4. what happens when the game is not over yet? 
 - Sub tests for 3 and 4 would include things like making sure that each player gets a turn, the board is displayed before someone makes a move, that the player/move function is called to get a player move. *note* `with-redefs`ing the human move or computer move is tying this down to a human and computer player. It now defeats the purpose of abstracting away "Player"